roles/StackStorm.st2/tasks/main.yml

```diff
@@ -100,3 +100,15 @@
     state: started
   loop: "{{ st2_services }}"
   tags: st2
+
+# https://github.com/ansible/ansible/issues/41313#issuecomment-520891625
+- name: Flush handlers to prepare to run st2 there are packs to install
+  include_tasks: flush_handlers.yml
+  when: st2_packs is defined
+
+- name: Install ST2 add-on Packs
+  import_tasks: packs.yml
+  when: st2_packs is defined and
```

Comment:
`st2_packs` is always defined from default values.

Would be better to check if st2_packs list size is > 0